ibm_concert/cert_renewal/send_cert_data.yml

```diff
@@ -0,0 +1,186 @@
+###############################################################################
+# © Copyright IBM Corporation 2024
+###############################################################################
+
+- hosts: all
+  collections:
```

Comment:
similar comment to another regarding using FQCN

![image](https://github.com/user-attachments/assets/1ad90976-d04e-426d-9b12-2ef3d0bb9105)

Reply:
will address in a new PR